LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Add the mathematical equation for each constraint for readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done</p>